lib/runner.js

```diff
@@ -158,14 +158,15 @@ Runner.prototype._addTestResult = function (test) {
 		this.stats.failCount++;
 	}
 
-	this.results.push({
+	var props = {
 		duration: test.duration,
 		title: test.title,
-		error: test.assertError,
+		err: test.assertError,
 		type: test.type
-	});
+	};
 
-	this.emit('test', test.assertError, test.title, test.duration, test.type);
+	this.results.push(props);
+	this.emit('test', test.assertError, props);
```

Comment:
To conform to errback style. However, since it's the event, not a callback, I can skip it.